.github/actions/bouf/action.yaml

```
@@ -0,0 +1,127 @@
+        
+        if ((Get-FileHash *windows*.zip -Algorithm SHA256).Hash -ne "$env:BOUF_HASH") {
+          throw "bouf hash does not match."
+        }
+        if ((Get-FileHash *nsis*.zip -Algorithm SHA256).Hash -ne "$env:BOUF_NSIS_HASH") {
+          throw "NSIS package hash does not match."
+        }
+        
+        Expand-Archive -Path *windows*.zip -DestinationPath bin
+        Expand-Archive -Path *nsis*.zip -DestinationPath nsis
+
+    - name: Download Google CNG Provider
+      shell: pwsh
+      env:
+        CNG_TAG: 'cng-v0.9'
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        # Download Google CNG provider release from github
+        . $env:GITHUB_ACTION_PATH\Ensure-Location.ps1
+        . $env:GITHUB_ACTION_PATH\Invoke-External.ps1
+        Ensure-Location gcng
+        
+        Invoke-External gh release download "$env:CNG_TAG" -R "GoogleCloudPlatform/kms-integrations" -p "*amd64.zip"
+        Expand-Archive -Path *.zip
+        $sigPath = Get-ChildItem *.sig -Recurse
+        $msiPath = Get-ChildItem *.msi -Recurse
+        # Verify digital signature against Google's public key
+        Invoke-External openssl dgst -sha384 -verify "$env:GITHUB_ACTION_PATH/cng-release-signing-key.pem" -signature $sigPath $msiPath
+        # Finally, install the CNG provider
+        Invoke-External msiexec /i $msiPath /qn /norestart
+
+    - name: Install rclone and pandoc
+      shell: pwsh
+      run: choco install pandoc rclone --no-progress -y
+
+    - name: Prepare Release Notes
+      shell: pwsh
+      run: |
+        # Release notes are just the tag body on Windows
+        cd repo
+        git tag -l --format='%(contents:body)' ${{ inputs.version }} > "${{ github.workspace }}/notes.rst"
+    
+    - name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        workload_identity_provider: ${{ inputs.gcpWorkloadIdentityProvider }}
+        service_account: ${{ inputs.serviceAccountName }}
+
+    - name: Download old builds
+      shell: pwsh
+      env:
+        RCLONE_DRIVE_ENV_AUTH: 'true'
+        RCLONE_DRIVE_SHARED_WITH_ME: 'true'
+      run: |
+        rclone copy --transfers 40 :drive:old_builds "${{ github.workspace }}/old_builds"
+
+    - name: Run bouf
+      shell: pwsh
+      env:
+        UPDATER_PRIVATE_KEY: ${{ inputs.updaterPrivateKey }}
+      run: |
+        $boufArgs = @(
+           "--config",     "$env:GITHUB_ACTION_PATH/config.toml",
+           "--version",    "${{ inputs.version }}"
+           "--branch",     "${{ inputs.channel }}"
+           "--notes-file", "${{ github.workspace }}/notes.rst"
+           "-i",           "${{ github.workspace }}/build"
+           "-p",           "${{ github.workspace }}/old_builds"
+           "-o",           "${{ github.workspace }}/output"
+        )
+        &"${{ github.workspace }}\bouf\bin\bouf.exe" @boufArgs
```

```suggestion
        Invoke-External "${{ github.workspace }}\bouf\bin\bouf.exe" @boufArgs
```

If not, then at least:
```
& "${{ github.workspace }}\bouf\bin\bouf.exe" @boufArgs
```